generator/src/main/java/org/corfudb/generator/distributions/DataSet.java

```diff
@@ -0,0 +1,46 @@
+package org.corfudb.generator.distributions;
+
+import java.util.ArrayList;
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collections'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)